assets/js/util/test/prepareSecondsForDisplay.js

```diff
@@ -1,41 +0,0 @@
-/**
- * Internal dependencies
- */
-import { prepareSecondsForDisplay } from '../';
-
-const valuesToTest = [
-	[
-		65,
-		'1m 5s',
-	],
-	[
-		125,
-		'2m 5s',
-	],
-	[
-		35,
-		'35s',
-	],
-	[
-		60,
-		'1m',
-	],
-	[
-		65,
-		'1m 5s',
-	],
-	[
-		( 60 * 60 * 3 ) + ( 60 * 5 ) + 12,
-		'3h 5m 12s',
-	],
-	[
-		( 60 * 60 * 7 ) + ( 60 * 2 ) + 42,
-		'7h 2m 42s',
-	],
```

Comment:
Let's also include/merge these cases with the new tests. There are a few cases here which are important to keep such as `35`, `60` and `65` (although we don't need to include `65` twice like it is here 😄 ). It would also be good to ensure we're not breaking anything in the process.